install-dev/data/xml/state.xml

```diff
@@ -980,6 +980,93 @@
     </state>
     <state id="AU-WA" id_country="AU" id_zone="Oceania" iso_code="WA" tax_behavior="0" active="1">
       <name>Western Australia</name>
-    </state>  
+    </state>
+    <state id="IN-AP" id_country="IN" id_zone="Asia" iso_code="AP" tax_behavior="0" active="1">
+      <name>Andhra Pradesh</name>
+    </state>
+    <state id="IN-AR" id_country="IN" id_zone="Asia" iso_code="AR" tax_behavior="0" active="1">
+      <name>Arunachal Pradesh</name>
+    </state>
+    <state id="IN-AS" id_country="IN" id_zone="Asia" iso_code="AS" tax_behavior="0" active="1">
+      <name>Assam</name>
+    </state>
+    <state id="IN-BR" id_country="IN" id_zone="Asia" iso_code="BR" tax_behavior="0" active="1">
+      <name>Bihar</name>
+    </state>
+    <state id="IN-CT" id_country="IN" id_zone="Asia" iso_code="CT" tax_behavior="0" active="1">
+      <name>Chhattisgarh</name>
+    </state>
+    <state id="IN-GA" id_country="IN" id_zone="Asia" iso_code="GA" tax_behavior="0" active="1">
+      <name>Goa</name>
+    </state>
+    <state id="IN-GJ" id_country="IN" id_zone="Asia" iso_code="GJ" tax_behavior="0" active="1">
+      <name>Gujarat</name>
+    </state>
+    <state id="IN-HR" id_country="IN" id_zone="Asia" iso_code="HR" tax_behavior="0" active="1">
+      <name>Haryana</name>
+    </state>
+    <state id="IN-HP" id_country="IN" id_zone="Asia" iso_code="HP" tax_behavior="0" active="1">
+      <name>Himachal Pradesh</name>
+    </state>
+    <state id="IN-JK" id_country="IN" id_zone="Asia" iso_code="JK" tax_behavior="0" active="1">
+      <name>Jammu and Kashmir</name>
+    </state>
```

Comment:
You can delete it, cf. [here](https://github.com/PrestaShop/PrestaShop/issues/15234#issuecomment-548381827). Otherwise, everything looks okay. ;-)

Reply:
Or perhaps deactivate it by writing _0_ instead of _1_?

Reply:
Removed.